myproject/src/components/Dashboard.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="container">
+    <img alt="Vue logo" src="../assets/logo.png">
+    <br>
+    <h3><span>{{currentUser}}さんようこそ!</span>残高: {{wallet}}</h3>
```

Comment:
マスタッシュの場合には内側の`{}`の両内側にはスペースをあけましょう。
```suggestion
    <h3><span>{{ currentUser }}さんようこそ!</span>残高: {{wallet}}</h3>
```